web/package.json

```diff
@@ -37,6 +37,7 @@
     "react": "16.12.0",
     "react-dom": "16.12.0",
     "react-hook-form": "^6.8.4",
+    "react-hooks-global-state": "^1.0.1",
```

Comment:
At first, I wanted to pass fetched data using react-router state, but it does not work great on firefox using large data like ours. https://github.com/ReactTraining/react-router/issues/7108

I opted for this library because it seemed really simple. However, maybe React's context would have been better: https://reactjs.org/docs/context.html.